invenio_webhooks/version.py

```diff
@@ -30,4 +30,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = "1.0.0a2"
+__version__ = "1.0.0a3.dev20160623"
```

Comment:
BTW note that we only have the tag 1.0.0a1 push out... there is no 1.0.0a2 tag. CC @lnielsen who released `1.0.0a2`: do you have the tag existing locally? Can you push out?